tests/Integration/Behaviour/Features/Context/LegacyProductFeatureContext.php

```diff
@@ -265,7 +265,7 @@ protected function createProduct($productName, $price, $productQuantity)
         $this->addProduct($product);
 
         // Shared Storage
-        SharedStorage::getStorage()->set($productName, $this->getProductWithName($productName)->id);
+        SharedStorage::getStorage()->set($productName, (int) $this->getProductWithName($productName)->id);
```

Comment:
why casting value instead defining return type for the method ?

Reply:
Return type is already defined as `Product` which is a legacy ObjectModel in which I'm not 100% confident with the correct typing of each field